loop.py

```diff
@@ -59,6 +60,41 @@ def compute_unfairness(A, R):
     return np.abs(np.asarray(A) - np.asarray(R))
 
 
+def compute_max_unfairness(A, R, k):
+    min_value = sys.maxsize
```

Comment:
min = max

Reply:
min_value here represents the lowest value found in the list with unfairness, which (as I understand it at least) is the item with the largest attention deficit.